public/style.css

```diff
@@ -326,11 +326,10 @@ body {
   z-index: -10;
   overflow: auto;
   background-color: rgba(0, 0, 0, 0.08);
-  position: fixed;
+  position: sticky;
   top: 80px;
-  right: 20%;
-  height: 0%;
-  width: 23%;
+  margin: auto;
+  width: 60%;
   transition: .5s;
```

Comment:
The initial height of container should be zero. Right? Everything else looks fine.

Reply:
@LalitNM  yes initial height should be zero.